components/pricing_modal/content.tsx

```diff
@@ -139,6 +155,25 @@ function Content(props: ContentProps) {
         formatMessage({id: 'admin.billing.subscription.planDetails.features.mfa', defaultMessage: 'Multi-Factor Authentication (MFA)'}),
     ];
 
+    // Default professional price
+    const [professionalPrice, setProfessionalPrice] = useState(10);
```

Comment:
dont have much context here on the pricing but it would be good idea to extract this default pricing out to a constants for easy discoverability later on. Less magic numbers 😆 
thinking on these we should have a look at this https://eslint.org/docs/latest/rules/no-magic-numbers

Reply:
Alright, I've made it a new constant in its current file. It doesn't have to be in another file right?